### PR TITLE
Added new var "course"

### PR DESCRIPTION
**Pull request by RanbirAulakh**
To work with any Alan’s class.

### CODE REVIEW

File: COPADS.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -99,7 +99,7 @@
 				TargetAttributes = {
 					DC745E661DAAACA5003AA4CB = {
 						CreatedOnToolsVersion = 8.1;
-						DevelopmentTeam = W7GJMYD5A4;
+						DevelopmentTeam = XCR75EBVVN;
```
Comment (harlanhaskins) [inline COPADS.xcodeproj/project.pbxproj]: Can we revert this? I don't want to change the development team.

**Review by harlanhaskins — COMMENTED** [review]
Thanks, Ranbir! I'll merge this when the dev team is changed!

**Comment by RanbirAulakh** [comment]
Please check my latest commit!

**Comment by harlanhaskins** [comment]
Looks good to me! :shipit:

**Comment by harlanhaskins** [comment]
Thank you!

**Pull request merged**